domain-mapping/ui/Components/DomainRow.js

```diff
@@ -19,6 +20,27 @@ class DomainRow extends React.Component {
 		this.props.update( data );
 	};
 
+  /**
+   * Handle converting domains between secondary to media domains and vice versa.
+   *
+   * @param {Object} event
+   */
+  handleConvert = ( event ) => {
+	event.preventDefault();
+
+	const data = { ...this.props.domain };
+
+	if ( 1 === data.type ) {
+		/** Convert media domain to secondary domain. */
+		data.type = 2;
+	} else if ( 2 === data.type ) {
+		/** Convert media domain to secondary domain. */
+		data.type = 1;
+	}
```

Comment:
I'd suggest considering implementing enums in this way:
https://stackoverflow.com/questions/54436750/where-to-put-enums-in-my-react-application

Reply:
Good suggestion - thanks and changed to be Enum-based.